### ISSUE TITLE
Resolume module crashes on add

### ISSUE THREAD

**Issue by k0mario** [bitfocus/companion-module-resolume-arena#79]
Module crashes in Companion versions 3.3.0 and 3.3.1. Immediately after adding it says "Crashed" and "Connection config unavailable". Pannel with connection settings doesn't appear.
error: Failed to init: Error: Call timed out Error: Call timed out
    at t.IpcWrapper.sendWithCb (C:\Program Files\Companion\resources\main.js:2:2780924)
    at R.init (C:\Program Files\Companion\resources\main.js:2:3213058)
    at l.s (C:\Program Files\Companion\resources\main.js:2:3224514)
    at l.emit (node:events:517:28)
    at l.emit (node:domain:489:12)
    at ChildProcess.<anonymous> (C:\Program Files\Companion\resources\main.js:2:2767921)
    at ChildProcess.emit (node:events:517:28)
    at ChildProcess.emit (node:domain:489:12)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)

**Comment by 4xel** [comment]
can not reproduce, is this still happening?

**Comment by TimPeterson** [comment]
This happened for me. I had a configuration working properly, referencing Resolume using the local IP address of the same computer (not the loopback address). I then moved the computer to a different network with a different subnet, and the module crashed immediately. I upgraded to 3.4.1 with no change. Since there was no way to edit the configuration of a crashing module, I had to export the config, edit the JSON manually to set the new IP address, and import. After that, it worked fine.
